logstash-core/lib/logstash/config/pipelines_info.rb

```diff
@@ -93,11 +91,19 @@ def self.format_pipeline_vertex_section_stats(stats, pipeline)
 
           acc
         end
-        
-        acc << {
+        if LogStash::PluginMetadata.exists?(plugin_id.to_s)
```

Comment:
nitpick: splitting up the building of `cluster_uuid` variable and its use is a bit harder to follow than it needs to be. I think that the following is effectively identical, but is a little easier to follow.

~~~ ruby
        segment = {
          :id => plugin_id,
          :pipeline_ephemeral_id => pipeline.ephemeral_id
        }

        if LogStash::PluginMetadata.exists?(plugin_id.to_s)
          plugin_metadata = LogStash::PluginMetadata.for_plugin(plugin_id.to_s)
          cluster_uuid = plugin_metadata&.get(:cluster_uuid)
          segment[:cluster_uuid] = cluster_uuid unless cluster_uuid.nil?
        end

        acc << segment.merge(segmented)
~~~

Reply:
Thanks! I made this change as recommended.